inc/customizer/options/typography.php

```diff
@@ -28,26 +28,75 @@ class Typography extends Base_Customizer {
 	 */
 	public function add_controls() {
 		$this->sections_typography();
+		$this->controls_font_pairs();
 		$this->controls_typography_general();
 		$this->controls_typography_headings();
 		$this->controls_typography_blog();
 		$this->section_extra();
 	}
 
+	/**
+	 * Add controls for font pair section
+	 *
+	 * @return void
+	 */
+	private function controls_font_pairs() {
+		/**
+		 * Filters the font pairs that are available inside Customizer.
+		 *
+		 * @param array $pairs The font pairs array.
+		 *
+		 * @since 3.5.0
+		 */
+		$pairs = apply_filters( 'neve_font_pairings', Mods::get( Config::MODS_TPOGRAPHY_FONT_PAIRS, Config::$typography_default_pairs ) );
+
+		/**
+		 * Font Pairs Control
+		 */
+		$this->add_control(
+			new Control(
+				Config::MODS_TPOGRAPHY_FONT_PAIRS,
+				[
+					'transport'         => $this->selective_refresh,
+					'sanitize_callback' => 'sanitize_text_field',
+					'default'           => $pairs,
+				],
+				array(
+					'input_attrs' => array(
+						'pairs'       => $pairs,
+						'description' => [
+							'text' => __( 'Choose Font family presets for your Headings and Text.', 'neve' ),
+							'link' => apply_filters( 'neve_external_link', 'https://docs.themeisle.com/article/1340-neve-typography', esc_html__( 'Learn more', 'neve' ) ),
+						],
+					),
+					'label'       => esc_html__( 'Font pairings', 'neve' ),
```

Comment:
@JohnPixle @mghenciu @abaicus @preda-bogdan how about if we change this to `Customize Fonts` to avoid adding new strings?

Reply:
It's worth changing it to avoid adding the new strings.